diff --git a/bin/setup.js b/bin/setup.js
index 3a7c277..24bb22d 100644
--- a/bin/setup.js
+++ b/bin/setup.js
@@ -28,6 +28,9 @@ console.log(`
   The ${chalk.green("Content Preview API Token")}
     will be used to show not published data in your development environment.
 
+  The ${chalk.green("Author ID")}
+    will be used to filter blog posts by a specific author.
+
   Ready? Let's do it! 🎉
 `);
 
@@ -54,15 +57,24 @@ const questions = [
       !process.env.CONTENTFUL_DELIVERY_TOKEN,
     message: "Your Content Delivery API access token",
   },
+  {
+    name: "authorId",
+    message: "Your Author ID",
+    when: !argv.authorId && !process.env.CONTENTFUL_AUTHOR_ID,
+    validate: (input) =>
+      /^[a-z0-9]{12}$/i.test(input) ||
+      "Author ID must be 12 upper/lowercase alpha-numeric characters",
+  },
 ];
 
 inquirer
   .prompt(questions)
-  .then(({ spaceId, managementToken, accessToken }) => {
+  .then(({ spaceId, managementToken, accessToken, authorId }) => {
     const {
       CONTENTFUL_SPACE_ID,
       CONTENTFUL_ACCESS_TOKEN,
       CONTENTFUL_DELIVERY_TOKEN,
+      CONTENTFUL_AUTHOR_ID,
     } = process.env;
 
     // env vars are given precedence followed by args provided to the setup
@@ -81,6 +93,8 @@ inquirer
       argv.deliveryToken ||
       accessToken;
 
+    authorId = CONTENTFUL_AUTHOR_ID || argv.authorId || authorId;
+
     console.log("Writing config file...");
     const configFiles = [`.env.development`, `.env.production`].map((file) =>
       path.join(__dirname, "..", file)
@@ -93,6 +107,7 @@ inquirer
         `# Do NOT commit this file to source control`,
         `CONTENTFUL_SPACE_ID='${spaceId}'`,
         `CONTENTFUL_ACCESS_TOKEN='${accessToken}'`,
+        `CONTENTFUL_AUTHOR_ID=${authorId}`
       ].join("\n") + "\n";
 
     configFiles.forEach((file) => {
